linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/ignition/shared.yml

```diff
@@ -0,0 +1,18 @@
+        filesystem: root
+        mode: 0644
+        path: /etc/bashrc
```

Comment:
Please, make sure the files end  with a newline character. Note the red marks in the files.

Reply:
done, thanks